frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -219,6 +220,7 @@ function SurveyCompletionConditions(): JSX.Element {
                 </LemonField>
             )}
             <SurveyRepeatSchedule />
+            <SurveyResponsesCollection />
```

Comment:
I think the feature flag can be checked here instead within the component, that's what other components do at least, a bit of perf improvement